src/main/java/org/opensearch/security/transport/SecurityInterceptor.java

```diff
@@ -86,13 +87,13 @@ public class SecurityInterceptor {
     private final SSLConfig SSLConfig;
 
     public SecurityInterceptor(final Settings settings,
-            final ThreadPool threadPool, final BackendRegistry backendRegistry,
-            final AuditLog auditLog, final PrincipalExtractor principalExtractor,
-            final InterClusterRequestEvaluator requestEvalProvider,
-            final ClusterService cs,
-            final SslExceptionHandler sslExceptionHandler,
-            final ClusterInfoHolder clusterInfoHolder,
-            final SSLConfig SSLConfig) {
+                               final ThreadPool threadPool, final BackendRegistry backendRegistry,
```

Comment:
Could you revert the format changes, it makes it harder to see what is actually changed?

Reply:
interesting, I didn;t make this change (I verified using git blame locally)